src/Microsoft.AspNetCore.Razor.Design/Microsoft.AspNetCore.Razor.Design.csproj

```diff
@@ -20,84 +18,27 @@
     We then need to include the output of those projects in our output directory (where it will be used
     by tests) and in the nukpg.
   -->
-
-  <!-- This is the tasks project that needs to be included in the package. -->
-  <ItemGroup>
-    <TaskProject Include="..\Microsoft.AspNetCore.Razor.Tasks\Microsoft.AspNetCore.Razor.Tasks.csproj" />
-  </ItemGroup>
-
   <!-- These are tools that need to be included in the package. -->
-  <ItemGroup>
-    <ToolProject Include="..\Microsoft.AspNetCore.Razor.Tools\Microsoft.AspNetCore.Razor.Tools.csproj" />
-  </ItemGroup>
-
-  <!-- Using explicit SDK imports here because the default way conflicts with the AfterBuild target -->
-  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <GenerateNuspecDependsOn>_BuildDependencyProjects;$(GenerateNuspecDependsOn)</GenerateNuspecDependsOn>
-    <BuildDependsOn>_BuildDependencyProjects;$(BuildDependsOn)</BuildDependsOn>
+    <ToolProject>..\Microsoft.AspNetCore.Razor.Tools\Microsoft.AspNetCore.Razor.Tools.csproj</ToolProject>
+    <TaskProject>Microsoft.AspNetCore.Razor.Tasks</TaskProject>
```

Comment:
Include the path to the csproj here to keep it consistent.

Reply:
I need the name though.